Exiled.Permissions/Features/Group.cs

```diff
@@ -31,5 +38,25 @@ public class Group
         /// Gets or sets the group permissions.
         /// </summary>
         public List<string> Permissions { get; set; } = new List<string>();
+
+        /// <summary>
+        /// Gets the combined permissions of the group plus all inherited groups.
+        /// </summary>
+        [YamlIgnore]
+        public List<string> CombinedPermissions
```

Comment:
Shouldn't we set this Property only when `Permissions.Reload()` get called?

Reply:
The current solution essentially does that, but it might be better to explicitly do it.